braid/braid_status.h

```diff
@@ -482,6 +510,16 @@ braid_StatusSetSize(braid_Status status,                   /**< structure contai
                     braid_Real   size                      /**< input, size of the send buffer */
                     );
 
+/**
```

Comment:
Maybe change comment to  ?
"""
Set the size of the buffer for basis vectors.  If 
set by user, the send buffer will allocate "size" bytes 
of space for each basis vector.  If not, BufSize is used 
for the size of each basis vector.
"""